lib/document.ex

```diff
@@ -18,16 +18,16 @@ defmodule Prismic.Document do
   @type fragment :: any()
   @type t :: %__MODULE__{
           id: String.t(),
-          uid: String.t(),
+          uid: String.t() | nil,
           type: String.t(),
           href: String.t(),
           tags: [String.t()],
           slugs: [String.t()],
           first_publication_date: DateTime.t() | nil,
           last_publication_date: DateTime.t() | nil,
           lang: String.t(),
-          alternate_languages: [String.t()],
-          fragments: [fragment]
+          alternate_languages: %{optional(String.t()) => AlternateLanguage.t()},
+          fragments: map
```

Comment:
Shouldn't it be `map()`?
```suggestion
          fragments: map()
```

Reply:
The parens aren't necessary, so we've been tending to exclude them from our Elixir codebases.